builtin/logical/ssh/path_config_ca.go

```diff
@@ -20,6 +24,10 @@ func pathConfigCA(b *backend) *framework.Path {
 				Type:        framework.TypeString,
 				Description: `Public half of the SSH key that will be used to sign certificates.`,
 			},
+			"generate_signing_key": &framework.FieldSchema{
+				Type:        framework.TypeBool,
+				Description: `Generate SSH key pair internally rather than use the private_key and public_key fields.`,
```

Comment:
I was initially going to go with true, but that would mean the backend would be initialised by `vault write -f ssh/config/ca` as it errors out without any data